tex/gloss-marathi.ldf

```diff
@@ -87,5 +88,24 @@ and may look very wrong.}
    }%
    \def\today{\marathinumber\day\space\marathimonth\space\marathinumber\year}%
 }
+\renewcommand{\labelitemii}{$\rightarrow$}
```

Comment:
This change needs to go to `\blockextrasmarathi` (and needs to be undone in `\noextrasmarathi`).

I.e.

```
\let\xpg@orig@labelitemii\labelitemii

\def\blockextrasmarathi{%
   ...
   \renewcommand{\labelitemii}{$\rightarrow$}
}

\def\noextrasmarathi{%
   ...
   \let\labelitemii\xpg@orig@labelitemii
}
```

Reply:
Done!